### PR TITLE
removes the usage of pg_worker_list.conf

### PR DESCRIPTION
**Pull request by furkansahin**
Needs to be reviewed together with the [PR in workerlist-gen repository](https://github.com/citusdata/workerlist-gen/pull/2).

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,15 +3,18 @@ version: '2'
 services:
```
Comment (jasonmp85) [inline docker-compose.yml]: Can you bump the version to version `'3'`?

File: docker-compose.yml [hunk 2]
```diff
@@ -3,15 +3,18 @@ version: '2'
...
   master:
     container_name: 'citus_master'
-    image: 'citusdata/citus:6.2.3'
+    image: 'citusdata/citus:7.0.0'
     ports: ['5432:5432']
     labels: ['com.citusdata.role=Master']
     volumes: ['/var/run/postgresql']
```
Comment (jasonmp85) [inline docker-compose.yml]: This volume is no longer needed.

File: docker-compose.yml [hunk 3]
```diff
@@ -3,15 +3,18 @@ version: '2'
 services:
   master:
     container_name: 'citus_master'
-    image: 'citusdata/citus:6.2.3'
+    image: 'citusdata/citus:7.0.0'
     ports: ['5432:5432']
```
Comment (jasonmp85) [inline docker-compose.yml]: I'm thinking about a better way of dealing with this (the port) to allow multiple compose clusters at once; will update tomorrow.

**Comment by jasonmp85** [comment]
(my branch with some of the above changes and a `HEALTHCHECK` is called `modernize`).

File: docker-compose.yml (hunk 3 above)
Comment (furkansahin) [reply docker-compose.yml]: Did you have a chance to look at this?

File: docker-compose.yml [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
 
 services:
   master:
     container_name: 'citus_master'
-    image: 'citusdata/citus:6.2.3'
+    image: 'citus:modern'
     ports: ['5432:5432']
     labels: ['com.citusdata.role=Master']
-    volumes: ['/var/run/postgresql']
+    environment:
```
Comment (jasonmp85) [inline docker-compose.yml]: These shouldn't be set here; they're included in the image if the user wants them but shouldn't be necessary for us. End-users can extend our image or apply environment variables if they wish to specify these, but our template shouldn't include them.

**Comment by jasonmp85** [comment]
This is a :shipit:. Gonna clean it up and merge (development in this repo is done in the `develop` branch, so I'll merge into that rather than into here).

**Pull request merged**